core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -108,6 +108,10 @@ public BigQuerySqlDialect(SqlDialect.Context context) {
             && !SqlTypeUtil.isNumeric(call.type);
   }
 
+  @Override public boolean supportsNestedAggregations() {
+    return false;
```

Comment:
LGTM.

I verified BigQuery by running a query and it does not support nested aggregation. BigQuery doc also does not mention the support of this feature (https://cloud.google.com/bigquery/docs/reference/standard-sql/aggregate_functions)

Reply:
Thanks for verifying @amaliujia. One minor thing: for non specific code related things it is better to add comments in the JIRA case. It is better for people who are going to look into the case in the future and it also creates less notifications.

Reply:
Gotcha. Thanks for your suggestion!

Reply:
Thank you @amaliujia